tests/test_copie.py

```diff
@@ -205,3 +208,100 @@ def test_valid_copier_config(_copier_config_file):
     test_check(result, "test_user_dir")
     test_check(result, "test_valid_copier_config")
     assert result.ret == 0
+
+
+class TestInclude:
```

Comment:
This is a very small lib and I was not using classes in my test on purpose to keep things simple. Can you remove the `class` object ?